### PR TITLE
mail-filter/afew: Add version bump 2.0.0 and 3.0.0

### PR DESCRIPTION
**Pull request by GuillaumeSeren**

### CODE REVIEW

File: mail-filter/afew/afew-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+		dev-python/dkimpy[${PYTHON_MULTI_USEDEP}]
+		net-mail/notmuch[python,${PYTHON_MULTI_USEDEP}]
+	')"
+DEPEND="
+	$(python_gen_cond_dep '
+		doc? ( dev-python/sphinx[${PYTHON_MULTI_USEDEP}] )
```
Comment (aidecoe) [inline mail-filter/afew/afew-2.0.0.ebuild]: Why is sphinx conditional dependency while in python_compile_all you're using build_sphinx unconditional anyway?

File: mail-filter/afew/afew-3.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+		dev-python/dkimpy[${PYTHON_MULTI_USEDEP}]
+		net-mail/notmuch[python,${PYTHON_MULTI_USEDEP}]
+	')"
+DEPEND="
+	$(python_gen_cond_dep '
+		doc? ( dev-python/sphinx[${PYTHON_MULTI_USEDEP}] )
```
Comment (aidecoe) [inline mail-filter/afew/afew-3.0.0.ebuild]: Same here

Comment (GuillaumeSeren) [reply mail-filter/afew/afew-3.0.0.ebuild]: That is true have I removed this conditionnal.

File: mail-filter/afew/afew-2.0.0.ebuild (hunk 1 above)
Comment (GuillaumeSeren) [reply mail-filter/afew/afew-2.0.0.ebuild]: Aggreed I removed it.

**Comment by GuillaumeSeren** [comment]
merged

**Pull request closed without merging**